python/lsst/rubintv/production/__init__.py

```diff
@@ -27,3 +27,5 @@
 from .monitorPlotting import *
 from .catchupService import *
 from .channels import *
+from .watchers import *
+from .metadataServers import *
```

Comment:
Does that need or even want doing? I left that out deliberately, as nobody should be instantiating those classes directly, and I expected any files which wanted stuff from them to be doing relative imports to get them. Is that bad or unwise?

Reply:
I think it might confuse Sphinx, if you cared about running that, and I think it might be worth a code comment, but I don't think it's bad if it's intentional.

Reply:
Oh, it was a passing thought, I'm happy to just add everything if that's the norm 👍